### DISCUSSION TITLE
[M-Z] (12 Febbraio 2019) Esercizio 1

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#437] [Esami & Esoneri M-Z]
## Traccia

<img width="583" alt="image" src="https://github.com/user-attachments/assets/792608e5-cfb5-4149-9514-4132434c2600">

**Comment by rimaout** [comment]
<img width="700" alt="image" src="https://github.com/user-attachments/assets/41bdb3b4-084f-45cc-9325-3a51536a4802">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20190212.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20190212.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/78f5391e-adb5-459f-a2f1-c2023d8f989f)
